build/terraform/gke/module.tf

```diff
@@ -88,6 +78,21 @@ variable "image_pull_secret" {
   default = ""
 }
 
+// Note: This is the number of gameserver nodes. The Agones module will automatically create an additional
```

Comment:
This lines was moved for easier diffs with `examples/terraform-submodules/gke/module.tf`